src/util/isEventEmitter.ts

```diff
@@ -0,0 +1,15 @@
+export type EventEmmitterLike1 = {
```

Comment:
If it's necessary to avoid compiler exception, I'm ok. In general principal would like encapsulate as much as we can, such as suggestions like avoid a separate isEventEmitter file.

BTW, (I know it's picky..) - any good suggestions of naming can replace xxx1, xxx2? @david-driscoll ?